app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -2312,6 +2312,38 @@ class ChatController(args: Bundle) :
                         clipboardManager.setPrimaryClip(clipData)
                         true
                     }
+                    R.id.action_mark_as_unread -> {
+                        val chatMessage = message as ChatMessage?
+                        ncApi!!.setChatReadMarker(
+                            credentials,
+                            ApiUtils.getUrlForSetChatReadMarker(
+                                1,
```

Comment:
@nickvergessen I hard-coded v1 API since calling it for v4 gives my a `404` return code.

Reply:
Chat endpoints are at API level 1 at the moment, so use:
https://github.com/nextcloud/talk-android/blob/3bc4c0c9838489277bf6c1fec6abae50a60f21a8/app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java#L173 with supported versions "1":
https://github.com/nextcloud/talk-android/blob/bb422fb11213192caabe290bb8065723f1bc631a/app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt#L1802

Reply:
Thanks :)

Reply:
Fixed via https://github.com/nextcloud/talk-android/pull/1752/commits/0ba210a0cae90883395eb8aeec190a063831a567